README.md

```diff
@@ -28,24 +28,28 @@ bugsnag:
 # Usage #
 After the installation the bundle works without any additional settings required, but you can tweak some settings.
 
-## Notify Stages ##
-You can set for which environments you want Bugsnag to get error reports. This is done with the notify_stages setting:
+## Enabled Stages ##
+You can set for which environments you want Bugsnag to be enabled. This is done through the enabled_stages setting:
 
 ```yml
 bugsnag:
-    notify_stages: [development, staging, production]
+    enabled_stages: [dev, prod, staging]
```

Comment:
I think we should keep this and the `notify_stages` below consistent. I suggest we use `[prod]` or `[prod,production]` as the default value (for backward compatibility) but use [development, staging, production] as the example.